src/pallavi/Assignment_16/Frequency.java

```diff
@@ -0,0 +1,28 @@
+package pallavi.Assignment_16;
+
+import java.util.Scanner;
+
+public class Frequency {
+
+	int count = 0;
```

Comment:
You should write this instance variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at instance. just trying to explain good coding practices.